### DISCUSSION TITLE
How to add superscript to the toolbar

### DISCUSSION THREAD

**Discussion by sulaimanhussain** [megahertz/react-simple-wysiwyg#42] [Q&A]
As the title suggests, I'm a little stuck and not sure how to go about this.

I'm just using the default configuration and got it working with no issues to output on the front end but unsure how I can extend it to add a superscript button

Thanks.

```
import { DefaultEditor } from 'react-simple-wysiwyg';

<div className="p-input__field">
      <DefaultEditor value={props.value} onBlur={props.onBlur} onChange={(event) => handleInputChange(props.name, props.type, event)} />
</div>
```

which outputs like this

![simple-wysiwyg](https://github.com/megahertz/react-simple-wysiwyg/assets/117170173/e6ce05c1-c06d-424f-a018-2414c2658265)

**Comment by megahertz** [comment]
You can find a custom button example in readme. Use [superscript](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand#superscript) command for that.

**Reply by sulaimanhussain** [reply]
Sweet, thank you so much for that!
